src/main/java/org/myrobotlab/service/LegacyServo.java

```diff
@@ -691,7 +691,7 @@ public void setInverted(boolean invert) {
   }
   
   @Override
-  public void unsetSpeed() {
+  public void disableSpeedControl() {
     log.info("disabling speed control");
     velocity = null;

```

Comment:
for consistency, if we're killing "velocity" this variable should be renamed to "speed".